src/Dapr.Actors.AspNetCore/ActorsServiceCollectionExtensions.cs

```diff
@@ -45,6 +46,16 @@ public static void AddActors(this IServiceCollection services, Action<ActorRunti
             services.TryAddSingleton<ActorRuntime>(s =>
             {   
                 var options = s.GetRequiredService<IOptions<ActorRuntimeOptions>>().Value;
+
+                //Replace the HttpEndpoint with an endpoint prioritizing IConfiguration
+                var configuration = s.GetService<IConfiguration>();
+                options.HttpEndpoint = options.HttpEndpoint != "http://127.0.0.1:3500"
```

Comment:
Should we be comparing against `"http://localhost:3500"`?  (This is one reason why I dislike the options pattern that sets values to an explicit default; options objects should have nullable properties that allow one to know whether they've been set.)

Reply:
Updated to reflect the spec - prefer whatever is provided in the options (made them support nullable values by default for HttpEndpoint and DaprApiToken) and failover to the configuration/envar if not provided.

Reply:
Yep - scrubbed this entirely in favor of the DaprDefaults implementation.